railties/test/generators/app_generator_test.rb

```diff
@@ -776,6 +776,17 @@ def test_quiet_option
     assert_empty output
   end
 
+  def test_quiet_option_quiets_rails_commands
+    generator = generator_class.new(
+      [File.join(destination_root, "myapp")], ["--quiet"], destination_root: destination_root
+    )
+
+    generator.stub :bundle_install?, -> { caller.grep(/run_hotwire/).any? } do
+      output = capture(:stdout) { generator.invoke_all }
+      assert_empty output
```

Comment:
```suggestion
      output = capture(:stdout) { run_generator }
      assert_empty output
      assert_empty @bundle_commands
```

How about this?

Reply:
I don't think that works, doing the following:

```suggestion
      output = capture(:stdout) { run_generator }
      assert_empty output
      assert_nil @bundle_commands
```

makes the test pass, but it also passes if I revert my `app_base.rb` code change, making this a test that passes all the time no matter what. 😬

Reply:
I mean, the fact that we need to stub `bundle_install?` at all is a smell to me.

I would rather we proxy a generator that create some noise in a command that we can explicitly test for.

There maybe a day when `run_hotwire` etc are removed, or could break in a separate PR and this test failing will be unrelated.

Sorry I don't have any concrete suggestions at the moment. I will try to come up with one though.